db/seeds/activities/stem_learning.rb

```diff
@@ -2031,30 +2043,38 @@
   activity.programmes = [secondary_certificate]
 end.save!
 
-Activity.find_or_create_by(stem_course_template_no: "5a945373-7d1a-ee11-8f6d-002248c6f9ce") do |activity|
+Activity.find_or_initialize_by(stem_course_template_no: "5a945373-7d1a-ee11-8f6d-002248c6f9ce").tap do |activity|
   activity.title = "Physical computing kits - KS1 BeeBots"
   activity.slug = "physical-computing-kits-ks1-beebots"
   activity.category = "face-to-face"
   activity.provider = "stem-learning"
   activity.stem_activity_code = "CP256"
-end
+  activity.credit = 15
 
-Activity.find_or_create_by(stem_course_template_no: "5dcbf737-121b-ee11-8f6d-002248c6f524") do |activity|
+  activity.programmes = [primary_certificate]
+end.save!
+
+Activity.find_or_initialize_by(stem_course_template_no: "5dcbf737-121b-ee11-8f6d-002248c6f524").tap do |activity|
   activity.title = "Physical computing kits - KS2 data loggers"
   activity.slug = "physical-computing-kits-ks2-data-loggers"
   activity.category = "face-to-face"
   activity.provider = "stem-learning"
   activity.stem_activity_code = "CP257"
-end
+  activity.credit = 15
+
+  activity.programmes = [primary_certificate]
+end.save!
 
-Activity.find_or_create_by(stem_course_template_no: "7b4a4c4b-a920-ee11-9966-002248c6f9ce") do |activity|
+Activity.find_or_initialize_by(stem_course_template_no: "7b4a4c4b-a920-ee11-9966-002248c6f9ce").tap do |activity|
   activity.title = "Computing Quality Framework – driving change within your school - short course"
   activity.slug = "computing-quality-framework--driving-change-within-your-school--short-course"
   activity.category = "face-to-face"
   activity.remote_delivered_cpd = true
   activity.provider = "stem-learning"
   activity.stem_activity_code = "CP403"
-end
+
+  activity.programmes = [primary_certificate]
```

Comment:
Could you add the number of the credits for this activity

Reply:
Good spot!